packages/accounts/src/light-account/actions/transferOwnership.ts

```diff
@@ -56,7 +56,7 @@ export const transferOwnership: <
   });
 
   if (waitForTxn) {
-    return client.waitForUserOperationTransaction(result);
+    return await client.waitForUserOperationTransaction(result);
```

Comment:
It's inefficient to await in a return. Just return the promise directly

Reply:
? then why above we r using await?

Reply:
What'd you mean? Before the code was just returning without the `await`. I'm saying `return await promise` is less efficient than `return promise` in an async method.

https://eslint.org/docs/latest/rules/no-return-await going off of this I'm confused now. It definitely used to be the case that doing `return await promise` was worse than `return promise` because of:

> Using return await inside an async function keeps the current function in the call stack until the Promise that is being awaited has resolved, at the cost of an extra microtask before resolving the outer Promise. return await can also be used in a try/catch statement to catch errors from another function that returns a Promise

Reply:
removed

Reply:
Ah this was out of date info, huh TIL https://github.com/eslint/eslint/issues/17345